### ISSUE TITLE
[DEVELOPING] Make the System User Friendly

### ISSUE THREAD

**Issue by MaramRadhi** [MaramRadhi/ProjectPlanning#6]
**Note For Team leader: ** 
Imran will be the team leader for this group. The group leader should make sure everyone in their group knows what they are doing and can provide extra resources for them. Team leader should also make sure everyone is working, if anyone isn't working team leader MUST report to the scrum masters.

**Comment by MaramRadhi** [comment]
_NOTE:_ 
Team Leaders should update this section after one week to show progress of the group and each member. Please mention if you accomplished everything you were supposed to in the given time and who did what below.  

**Task Description Action: **

**Comment by imrun10** [comment]
**Task Description Action: **
Huda has written the code needed for database connection and has written an API GET request to retrieve the rows in the course table however there is a bug we are trying to solve that stops that breaks the connection with the database every time the request is made. We feel it has to do with the different versions of node js we have and the configuration of how we connected the database.

Shahd did nothing but after the meeting, after I told her she still hadn't done her part she then later at night sent some templates however by that point there was no need for them.

I created a component for the cards that will hold courses on them that can be used on the front page. i also wrote code that will dynamically update the front page with all the courses that we receive from the backend. I used just a test array to test this out so no matter how many courses it will rearrange them dynamically additionally it uses the card component to display them. I also update the visuals of the front end to make it more user friendly
